stubs/waitress/waitress/adjustments.pyi

```diff
@@ -8,7 +8,7 @@ from .proxy_headers import PROXY_HEADERS as PROXY_HEADERS
 truthy: frozenset[Any]
 KNOWN_PROXY_HEADERS: frozenset[Any]
 
-def asbool(s: bool | str | int | None) -> bool: ...
+def asbool(s: str | bool | None) -> bool: ...
```

Comment:
This one is just incorrect: it doesn't take `int`. https://github.com/Pylons/waitress/blob/4b6b5832f3a30d82dc8cb359cf04d2a8edf6712b/src/waitress/adjustments.py#L30-L41

Reply:
It technically does, since it calls `str()` on its argument and then checks whether the result is `"1"` (among others).

Reply:
I clearly need a coffee; I misread the source code.

Reply:
Changes have been reverted :)